system/Database/SQLite3/Table.php

```diff
@@ -214,23 +214,14 @@ public function dropPrimaryKey(): Table
      *
      * @return Table
      */
-    public function dropForeignKey(string $column)
+    public function dropForeignKey(string $foreignName)
```

Comment:
The parameter name has been changed.
Is this a breaking change or not?

Reply:
I suppose it is.. especially for anyone who might of extended the class. I'll make mention in the change log.

The functionality of this in the Table class was very marginal. In a lot of cases it wasn't even working at all. See https://github.com/codeigniter4/CodeIgniter4/issues/6453

With the new structure of _foreignKeyData() things are starting to function much better.. especially SQLite.

Reply:
Okay, thank you.

We are discussing parameter name changing is a breaking change or not in our project.
But at least, we should add the change in the changelog.

Extended class can change the parameter name.
https://3v4l.org/l8Xf5